frontend/packages/erd-core/src/components/ERDRenderer/ErrorDisplay.module.css

```diff
@@ -1,8 +1,161 @@
-/* TODO: styling */
 .wrapper {
-  position: relative;
-  width: 100%;
+  font-family: var(----main-font, Inter);
```

Comment:
Maybe this one? 🤔

```suggestion
  font-family: var(---main-font, Inter);
```
![ss 2655](https://github.com/user-attachments/assets/32dcfff1-53fd-4760-9dd5-a0f27784271e)

Reply:
@FunamaYukina  sorry, `--` is maybe correct. I'll check now 🙇  🙇

Reply:
Oh, Sorry! My suggestion was incorrect.😵

Reply:
Thank you for the catch!